src/api/methods/createWorkspace.ts

```diff
@@ -0,0 +1,4 @@
+export interface CreateWorkspaceRequest {
+  /** Token used to get workspace configuration */
+  token: string;
```

Comment:
We probably need to add here an id or node selector for the user to chose where the workspace should be rendered.